spec/features/user_spec.rb

```diff
@@ -43,9 +73,18 @@
       expect(page).to have_content('User updated.')
     end
 
+    it 'Verifies users role can be edited' do
+      @users_page.change_users_role('John', 'Editor')
+      expect(page).to have_content('Changed roles.')
```

Comment:
it would also be good to check that real value is updated, since the message might be there, but value stays the same